webpack/assets/javascripts/react_app/routes/routes.js

```diff
@@ -1,3 +1,4 @@
 import HostWizard from './HostWizard';
+import Models from './Models';
 
-export const routes = [HostWizard];
+export const routes = [HostWizard, ...Models];
```

Comment:
just so i understand, the spread is for future nested routing? e.g. `/models/new`

Reply:
Yes, that is the intention.